docs/installguide/install_all.rst

```diff
@@ -163,9 +156,56 @@ See the :doc:`release notes <release_notes>` for critical upgrade information fo
 Configuration after installation or update
 __________________________________________
 
+Some explanation is included here:
+
+1. Choose weather you want to run KA Lite on boot or not. We recommend choosing yes, as it simplifies data management.
+If you choose not to, you must manually start KA lite every time.
+
+.. note::
+    Running KA Lite as different users creates different sets of data files, so it's recommended that you run KA Lite as the same user every time.
+
+.. image:: linux-install-startup.png
+  :class: screenshot
+
+2. If you chose to start on boot in the previous step, you will be prompted to choose the owner for the KA Lite server
+process. Generally the default value is ok.
+
+.. image:: linux-install-owner.png
+  :class: screenshot
+
+3. You will be asked to review your choices, and finally KA Lite will start automatically when installation is complete.
+
+
+.. tip::
+    If you want to receive automatic updates from online sources, you can
+    also use :ref:`ppa-installation`.
+
+
+.. _raspberry-pi-install:
+
+
 Every time you install or update KA Lite, you must run ``kalite manage setup`` command again to setup the database and download assessment items (video descriptions,
 exercises etc.).
 
+During the setup it will ask to download assessment.zip that has all exercises. This file is around 500MB and it will take time
+depending on the internet connection
+
+Use following command to start/stop KA-LITE:: 
+
+     # Starting KA-LITE
+     sudo service kalite start
+     # Stopping KA-LITE 
+     sudo service kalite stop

```

Comment:
Dashes are missing in both the above, should be `sudo service ka-lite <start/stop/restart>`